bot/src/utils/openai/client.py

```diff
@@ -93,13 +93,14 @@ async def _parse_completion_choices(self, response: OpenAICompletion) -> str:
 
     async def get_completions(self, text: str, max_tokens: int = 4000, temperature: float = 1.0) -> str:
         data = {
-            'model': 'gpt-3.5-turbo-instruct',
-            'prompt': text,
+            'model': 'gpt-4o-mini',
+            'messages': [{"role": "user", "content": text}],
             'max_tokens': max_tokens,
             'temperature': temperature,
         }
-        response = await self._make_request(self.Method.COMPLETIONS, data)
-        return await self._parse_completion_choices(OpenAICompletion(**response))
+        
+        response = await self._make_request(self.Method.CHAT_COMPLETIONS, data)
+    return await self.parse_chat_choices(OpenAIChatChoices(**response))
```

Comment:
so, did you try to run the code?

it seems there is still a tabulation problem, what ado you think?